README.md

```diff
@@ -65,13 +65,24 @@ const bot = new Wechaty({
   // ...
   puppetOptions: {
     launchOptions: {
-      executablePath: '<executablePath>',
+      executablePath: '<executablePath>'
       // ... others launchOptions, see: https://github.com/GoogleChrome/puppeteer/blob/v1.18.1/docs/api.md#puppeteerlaunchoptions
     }
   }
 });
 ```
 
+We use [stealth](https://www.npmjs.com/package/puppeteer-extra-plugin-stealth) to make puppeteer more like a normal browser, if you want to disabled it, just set the `WECHATY_PUPPET_PUPPETEER_EXTRA_DISABLED` environment variable to `true`. eg. `WECHATY_PUPPET_PUPPETEER_EXTRA_DISABLED=true && ts-node your-bot.ts`
+
+## puppetOptions
+
+| Option        |  value  | default value | description                                                                                                                 |
+| ------------- | :-----: | :-----------: | :-------------------------------------------------------------------------------------------------------------------------- |
+| endpoint      | string  |       -       | puppeteerlaunchoptions.executablePath                                                                                       |
+| head          | boolean |     true      | puppeteerlaunchoptions.headless                                                                                             |
+| launchOptions | object  |       -       | same to [puppeteerlaunchoptions](https://github.com/GoogleChrome/puppeteer/blob/v1.18.1/docs/api.md#puppeteerlaunchoptions) |
+| stealth       | boolean |     true      | use [stealth](https://www.npmjs.com/package/puppeteer-extra-plugin-stealth) or not                                          |
```

Comment:
Thanks for adding this options table.

After reviewing the table, I believe we should use `stealthless` instead of `stealth` because we should keep the default(absence) value to `false`.